ember-toucan-core/src/components/form/radio-group-field.hbs

```diff
@@ -3,6 +3,8 @@
   @hint={{@hint}}
   @error={{@error}}
   @isDisabled={{@isDisabled}}
+  aria-invalid={{if @error "true"}}
+  role="radiogroup"
```

Comment:
Based on https://adrianroselli.com/2022/02/support-for-marking-radio-buttons-required-invalid.html#R6

> This is valid HTML and ARIA. Please bear in mind that the [radiogroup](https://www.w3.org/TR/wai-aria-1.1/#radiogroup) role is strictly for cases where only one item in a set can be selected at a time (so not checkboxes).

So since we are radios here, adding the `role` should be fine! We can't replicate this with grouped-checkboxes though.

Reply:
Based on the example, doesn't `aria-required` need to be added in also?

Reply:
According to https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role#associated_wai-aria_roles_states_and_properties

> aria-required
> Indicates that one radio within the group must have aria-checked="true" set before the form may be submitted. The required state is specified on the radiogroup element rather than one of the radio elements, unlike when using HTML radio buttons where the [required](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required) attribute is set directly on one or more radio [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) elements.


I'm not sure! We require a `value` as part of https://github.com/CrowdStrike/ember-toucan-core/pull/74, so they can't get in a state where a value isn't provided/automatically set.  Those docs explicitly mention `aria-checked` as well, where we use the `checked` attribute, so I'm not super sure it applies.  Normally `aria-checked` is used when _not_ using semantic HTML for radios/checkboxes, but here we are.  What do you think?